examples/tilemap/main.rs

```diff
@@ -77,16 +77,16 @@ fn spawn_tiles(
 ) {
     for (_, tileset) in tilesets.iter() {
         let texture_handle = tileset.texture.clone();
-        let material_handle = materials.add(ColorMaterial::texture(texture_handle));
+        //let material_handle = materials.add(ColorMaterial::from(texture_handle));
         // The layer_settings method of Tileset is implemented in the "bevy_ecs_tilemap" feature.
-        let settings = tileset.layer_settings(UVec2::new(3, 3), UVec2::new(3, 3));
+        let settings = tileset.layer_settings(MapSize(3, 3), ChunkSize(3, 3));
 
         let (mut layer_builder, layer_entity) =
             LayerBuilder::<TileBundle>::new(&mut commands, settings, 0u16, 0u16);
 
         set_tiles(&mut layer_builder);
 
-        map_query.build_layer(&mut commands, layer_builder, material_handle);
+        map_query.build_layer(&mut commands, layer_builder, texture_handle);
```

Comment:
I changed these because build_layer doesn't accept ColorMaterial. I made this change quickly without understanding what the example is trying to do, I'll fix it soon